android/src/main/java/com/tailscale/ipn/ui/theme/Theme.kt

```diff
@@ -44,7 +44,7 @@ fun AppTheme(useDarkTheme: Boolean = isSystemInDarkTheme(), content: @Composable
           // margins in list items.
           bodyMedium = MaterialTheme.typography.bodyMedium.copy(fontSize = 16.sp))
 
-  val systemUiController = rememberSystemUiController()
+  @Suppress("deprecation") val systemUiController = rememberSystemUiController()
```

Comment:
can you create an issue/todo to use the new API? looks like the recommendation is WindowInsetsControllerCompat or Activity.enableEdgeToEdge per https://google.github.io/accompanist/systemuicontroller/